pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -70,6 +74,9 @@ def __init__(
         self.task_idx = None
         self.node_rank = 0
         self.num_processes = len(parallel_devices) if parallel_devices is not None else parallel_devices
+        self.ddp_comm_state = ddp_comm_state
+        self.ddp_comm_hook = ddp_comm_hook
+        self.ddp_comm_wrapper = ddp_comm_wrapper
```

Comment:
if these aren't meant to be accessed outside this class or child classes we can mark these as private
```suggestion
        self._ddp_comm_state = ddp_comm_state
        self._ddp_comm_hook = ddp_comm_hook
        self._ddp_comm_wrapper = ddp_comm_wrapper
```